docs/products/postgresql/howto/pgbouncer-stats-datadog.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Expose PgBouncer statistics to Datadog
+sidebar_label: PgBouncer stats in Datadog
+---
+
+Connect PgBouncer with Datadog to display [PgBouncer statistics](/docs/products/postgresql/howto/pgbouncer-stats) on the Datadog platform.
+
+## Prerequisites
+
+- Your service plan is Startup or higher.
+- [Datadog metrics integration](/docs/integrations/datadog/datadog-metrics) is
+  enabled on your service.
+- You applied outstanding service maintenance updates affecting the Datadog metrics
+  integration.
+
+## Expose PgBouncer statistics
+
+You can expose PgBouncer statistics for your Aiven for PostgreSQL service both if the
```

Comment:
I recommend to drop the terms "expose" and "statistics" in favor of "metrics" and "monitoring". I recommend the opening similar/consistent to the one in Database monitoring: "To use Datadog PgBouncer Monitoring with your Aiven for PostgreSQL® services, ...".

I am also not clear about the "regardless if this service already has a Datadog Metrics integration or not." part. Then, this begs the question of why Datadog metrics integration is a prerequise in the first place.
@0xlianhu could you please clarify?